### PR TITLE
Added jenkins smoke groovy for smoke testing

### PR DESCRIPTION
**Pull request by ganeshhubale**
Added smoke test file in automation plugin - https://gitlab.cee.redhat.com/insights-qe/iqe-migration-analytics-plugin/-/merge_requests/41

### CODE REVIEW

File: Jenkinsfile-smoke.groovy [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+@Library("github.com/RedHatInsights/insights-pipeline-lib@v3") _
+
+if (env.CHANGE_ID) {
+    execSmokeTest(
+        // the service-set/component for this app in e2e-deploy "buildfactory"
+        ocDeployerBuilderPath: "xavier/xavier-analytics",
+    
+        // the service-set/component for this app in e2e-deploy "templates"
+        ocDeployerComponentPath: "xavier/xavier-analytics",
```
Comment (mrizzi) [inline Jenkinsfile-smoke.groovy]: This must be changed to use the right one.
```suggestion
        ocDeployerComponentPath: "xavier/xavier-analytics-kieserver",
```

**Review by mrizzi — CHANGES_REQUESTED** [review]

File: Jenkinsfile-smoke.groovy (hunk 1 above)
Comment (ganeshhubale) [reply Jenkinsfile-smoke.groovy]: Agree! It comes from [here](https://github.com/RedHatInsights/e2e-deploy/tree/master/templates/xavier)

Comment (mrizzi) [reply Jenkinsfile-smoke.groovy]: Thanks

**Review by mrizzi — APPROVED** [review]
LGTM, thanks @ganeshhubale

**Pull request merged**